prisma/schema.prisma

```diff
@@ -34,18 +34,20 @@ model Password {
 }
 
 model Group {
-  id          String  @id @default(uuid())
+  id          String       @id @default(uuid())
   name        String
-  description String  @db.Text
+  description String       @db.Text
   imgUrl      String?
   imgAlt      String?
   events      Event[]
-  users       User[]
+  users       User_Group[]
 }
 
 model User_Group {
-  userId  String
-  groupId String
+  userId  String @unique
```

Comment:
I think if we make groupId and userId `@unique` it would mean one unique user can only have one group and vice versa.

```suggestion
  userId  String
```

Reply:
![image](https://github.com/social-plan-it/plan-it-social-web/assets/29219684/84f192a2-4276-42c7-97c8-a56c1a887d36)
removing @unique would cause prisma to complain